bin/job

```diff
@@ -71,15 +60,17 @@ transformVM() {
   local VMYAML=${DOMXML%.xml}.yaml
 
   # we support only one disk, it needs to be aligned with DNS-1123
-  PVCNAME=$(echo $SRC-disk-1 | sed -r 's/[_.]+/-/g')
-  VMNAME=$(echo $SRC | sed -r 's/[_.]+/-/g')
+  export PVCNAME=$(echo $SRC-disk-1 | sed -r 's/[_.]+/-/g')
+  export VMNAME=$(echo $SRC | sed -r 's/[_.]+/-/g')
 
   # get resources from vm xml
-  CORES=$(xmllint --xpath "/domain/vcpu/text()" $DOMXML)
-  MEMORY=$(xmllint --xpath "/domain/memory/text()" $DOMXML)
-  UNIT=$(xmllint --xpath "string(/domain/memory/@unit)" $DOMXML)
-  BUS=$(xmllint --xpath "string(/domain/devices/disk/target/@bus)" $DOMXML)
+  export CORES=$(xmllint --xpath "/domain/vcpu/text()" $DOMXML)
+  export MEMORY=$(xmllint --xpath "/domain/memory/text()" $DOMXML)
+  export UNIT=$(xmllint --xpath "string(/domain/memory/@unit)" $DOMXML)
+  export BUS=$(xmllint --xpath "string(/domain/devices/disk/target/@bus)" $DOMXML)
+}
```

Comment:
All of the above would go into a script called import-v2v.sh.  It would export all vars and call create-vm.sh which would be the implementation of createVM below (converted to pull info from env vars.

When I want to add import from url support I would:
 - write a script called import-url.sh which will create the pvc and export required vars
 - run the same create-vm.sh

We can move all scripts into this repo and copy them in using the Dockerfile.

Reply:
OK